x/blob/ante/ante_test.go

```diff
@@ -141,6 +145,48 @@ func TestPFBAnteHandler(t *testing.T) {
 	}
 }
 
+// TestMinGasPFBDecoratorWithMsgExec tests that the MinGasPFBDecorator rejects a
+// MsgExec that contains a MsgExec with a MsgPayForBlob where the MsgPayForBlob
+// gas cost is greater than the tx's gas limit.
+func TestMinGasPFBDecoratorWithMsgExec(t *testing.T) {
+	anteHandler := ante.NewMinGasPFBDecorator(mockBlobKeeper{})
+	txConfig := encoding.MakeConfig(app.ModuleEncodingRegisters...).TxConfig
+
+	// Create a context with a gas meter with a high gas limit
+	gasLimit := uint64(10000)
```

Comment:
doesn't a normal send usually use 70k?

Reply:
TBH I have no idea. Just found a random PFB on Celenium that uses 10 million gas so will increase this 

Ref: https://celenium.io/tx/6aa170e9f68fda9382cd35559117579042e6eab56df567ac78c1079e34c7707c?tab=messages

Reply:
https://github.com/celestiaorg/celestia-app/pull/4465/commits/9059f172185d9b44ade3c0d82148a7ad1c14bef9